scene/main/node.cpp

```diff
@@ -2816,7 +2816,9 @@ Node *Node::duplicate(int p_flags) const {
 	ERR_THREAD_GUARD_V(nullptr);
 	Node *dupe = _duplicate(p_flags);
 
-	if (dupe && (p_flags & DUPLICATE_SIGNALS)) {
+	ERR_FAIL_NULL_V_MSG(dupe, nullptr, "Failed to duplicate node (Node::_duplicate returned nullptr)");
```

Comment:
```suggestion
	ERR_FAIL_NULL_V_MSG(dupe, nullptr, "Failed to duplicate node.");
```
`_duplicate()` is an internal method. Its existence is irrelevant to the user. Though you could add node's name/class to the message, for more context.